README.md

```diff
@@ -28,19 +28,17 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 
 - Windows 11 or Windows 10 version 2004 (code name 20H1 / build number 19041) or newer.
 - Our installer will install the following items:
-   - [.NET 6.0.7 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/6.0#runtime-desktop-6.0.7) or a newer 6.0.x runtime.
+   - [.NET 6.0.7 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/6.0#runtime-desktop-6.0.7).
    - [Microsoft Edge WebView2 Runtime](https://go.microsoft.com/fwlink/p/?LinkId=2124703) bootstrapper. This will install the latest version.
    - [Microsoft Visual C++ Redistributable](https://docs.microsoft.com/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022) installer. This will install one of the latest versions available.
 
 ### Via GitHub with EXE [Recommended]
 
- [Microsoft PowerToys GitHub releases page][github-release-link], click on `Assets` at the bottom to show the files available in the release. Please use the appropriate the PowerToys installer that matches your machine's architecture.  For most people, it is `x64`.
+This is our preferred method. Please use the appropriate the PowerToys installer that matches your machine's architecture. For most people, it is `x64`.
```

Comment:
We should shift the Download links to the bottom link we do with the privacy and stuff

I would love if we could not directly reference the version number then as well.  Only arch. This will make updating a LOT easier and prevent mistakes.

Reply:
> I would love if we could not directly reference the version number then as well.

Sounds lovely, but how? As long as the installer files have a version number in them, there must be something&mdash;somewhere that points to the latest version file..?

Reply:
why not link to `github.com/.../releases/latest`?

Reply:
> We should shift the Download links to the bottom

Wait, what? Why? Installing requiers downloading, that kind of belongs together...